.git-blame-ignore-revs

```diff
@@ -1,2 +1,5 @@
 # To enable the use of this file in your local repo, type the following at the CLI:
 # git config blame.ignoreRevsFile .git-blame-ignore-revs
+
+# Run clang-format on all .cc and .h files in code/code/**
+1c508231f77464bf5807fba89ca72ab8c445ffc1
```

Comment:
Depends on how we merge, this hash might change and need updating. But let's see how it goes.

Reply:
Hm, yeah, hadn't thought about that. If it does, I'd probably have to modify the SHA in this file then merge that separately using the "merge" option instead "rebase", I assume?

Reply:
I vote not to worry about it today. Get the PR in, fix the hash in the next PR, if needed.